client/package.json

```diff
@@ -11,6 +11,9 @@
     "@babel/preset-react": "^7.25.9",
     "antd": "^5.22.1",
     "babel-loader": "^9.2.1",
+    "bootstrap-icons": "^1.11.3",
```

Comment:
[Optional] If you want, you can check the antd avatar icons. Not a neccessity but can remove extra package.

Reply:
No need to check again then I guess.